data/1.0/_index.md

```diff
@@ -34,24 +34,41 @@ garage.save(ferrari);
 
 ### New features, enhancements or additions
 
-* The CrudRepository feature
-* Allows pagination on repository with PageableRepository
+* Define the Repository pattern
+* Allows pagination on Repository
+* Enhance type safety with a StaticMetamodel
+* Define platform integrations with CDI, Persistence, NoSQL, Transactions, and Validation
+* Produce a TCK with the ability to run against the following matrix:
+  * Profiles: none, core, web, full
+  * Entities: any, persistence, nosql
 
+###  Removals, deprecations or backwards incompatible changes
+None - first release
 
 ### Minimum Java SE Version
-<!-- Specify the minimum required Java SE version for this specification -->
-**Java SE 17 or higher**
-
-
+* **Java SE 17 or higher**
 
 # Details
 
 * [Jakarta Data 1.0 Release Record](https://projects.eclipse.org/projects/ee4j.data/releases/1.0)
-* [Jakarta Data 1.0 Specification Document](./data-1.0.0-RC1.pdf) (PDF)
-* [Jakarta Data 1.0 Specification Document](./data-1.0.0-RC1.html) (HTML)
-* [Jakarta Data 1.0 Specification Javadoc](./apidocs)
+  * [Jakarta Data 1.0 Release Plan](https://projects.eclipse.org/projects/ee4j.data/releases/1.0/plan)
+  * [Jakarta EE Platform 11 Release Plan](https://projects.eclipse.org/projects/ee4j.jakartaee-platform/releases/11/plan)
+* [Jakarta Data 1.0 Specification Document](./jakarta-data-1.0.pdf) (PDF)
+  * [Jakarta Data 1.0 Specification Addendum](./jakarta-method-name-query-1.0.pdf) (PDF)
```

Comment:
```suggestion
  * [Jakarta Data 1.0 Specification Addendum](./jakarta-data-addendum-1.0.pdf) (PDF)
```
Why don't you merge this in the overall jakarta-data-1.0.pdf? Having multiple docs is confusing. Is this doc optional?

Reply:
The `method name query` addendum is separate because it is part of the specification we intend to deprecate in the future but is included in the 1.0 version of the specification to offer a migration path for Spring users.